src/main/java/bio/terra/janitor/service/cleanup/flight/GoogleBucketCleanupStep.java

```diff
@@ -0,0 +1,36 @@
+package bio.terra.janitor.service.cleanup.flight;
+
+import bio.terra.cloudres.common.ClientConfig;
+import bio.terra.cloudres.google.storage.StorageCow;
+import bio.terra.janitor.db.TrackedResource;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.Step;
+import bio.terra.stairway.StepResult;
+import com.google.cloud.storage.StorageOptions;
+
+public class GoogleBucketCleanupStep implements Step {
+  private final StorageCow storageCow;
+
+  public GoogleBucketCleanupStep() {
+    // Janitor only uses CRL Cows to delete resources. Cleanup is not needed.
+    this.storageCow =
+        new StorageCow(
+            ClientConfig.Builder.newBuilder().setClient("terra-crl-janitor").build(),
```

Comment:
Maybe doesn't have to be this PR, but I think long term this (and other cleanup steps) should take the ClientConfig as an argument to the constructor. Then, we should make a `@Component` that creates a ClientConfig and grab that during Flight construction.

Reply:
Done